### ISSUE TITLE
MemoryStore cannot handle Attribute API

### ISSUE THREAD

**Issue by JanStevens** [devmynd/jsonb_accessor#89]
Since the defaults are added using the Attribute API and it doesn't work with the Cache MemoryStore my specs blow up. I've made an issue on Rails end: https://github.com/rails/rails/issues/31216

The only way I could go around it (since my defaults are no procs) is by extracting the default to the attribute API

```ruby
jsonb_accessor :config, tshirt_sizes: [:string, array: true]
attribute :config, :jsonb, default: { tshirt_sizes: %w(S M L XL) }
```

Another way would be not to wrap the defaults in a lambda but only do it when the use also gave in a lambda

**Comment by stevenharman** [comment]
@JanStevens It looks like rails/rails#31216 was fixed in Rails 5.2.0. Can this issue be closed?

**Comment by JanStevens** [comment]
Yep, thanks for the follow-up
